sdk/monitor/microsoft-opentelemetry-exporter-azuremonitor/microsoft/opentelemetry/exporter/azuremonitor/export/trace/_exporter.py

```diff
@@ -0,0 +1,212 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+import json
```

Comment:
Why are we moving these files?

Reply:
File names starting with underscore indicate these only include internal code and are not exposed in documentation including apiview

Reply:
We don't want `AzureMonitorTraceExporter` exposed in the documentation?